src/sharedFramework/sharedFramework.proj

```diff
@@ -97,37 +100,25 @@
 
   <Target Name="GenerateRuntimeGraph">
     <PropertyGroup>
-      <runtimeGraphGeneratorName>DepsProcessor</runtimeGraphGeneratorName>
-      <runtimeGraphGeneratorProject>$(ProjectDir)tools-local/setuptools/independent/$(runtimeGraphGeneratorName)</runtimeGraphGeneratorProject>
-      <runtimeGraphGeneratorOutput>$(IntermediateOutputRootPath)setuptools/independent/$(runtimeGraphGeneratorName)</runtimeGraphGeneratorOutput>
-      <runtimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='Windows_NT'">win</runtimeGraphGeneratorRuntime>
-      <runtimeGraphGeneratorRuntime Condition="'$(OSGroup)'=='OSX'">osx</runtimeGraphGeneratorRuntime>
-      <runtimeGraphGeneratorRuntime Condition="'$(runtimeGraphGeneratorRuntime)'==''">linux</runtimeGraphGeneratorRuntime>      
+      <SharedFrameworkAssetsFile>$(SharedFrameworkSourceRoot)/obj/project.assets.json</SharedFrameworkAssetsFile>
```

Comment:
Agreed it ideally would. There are other places with this same issue already.  We should address this holistically.  https://github.com/dotnet/core-setup/issues/2322